src/steps/vulns/converters.ts

```diff
@@ -104,6 +109,11 @@ export function createVulnerabilityTargetEntities(
       displayName: vuln.TITLE,
       webLink: buildQualysGuardVulnWebLink(qualysHost, vuln.QID!),
       severityLevel: vuln.SEVERITY_LEVEL, // raw value, not normalized as it is on `Finding.numericSeverity`
+
+      qualysVulnType: vuln.VULN_TYPE,
+      qualysSolution: vuln.SOLUTION,
+      qualysDiscoveryRemote: vuln.DISCOVERY?.REMOTE,
+      qualysCategory: vuln.CATEGORY,
```

Comment:
@aiwilliams Chris seems to have implemented what you've asked for based on this ticket (https://jupiterone.atlassian.net/browse/INT-1294) but looking at it, I wouldn't expect these `qualys*` properties to exist at all on the `cve` entities, and I wouldn't expect them to be prefixed with `qualys` on the `qualys_vuln` entities. What was your intention?

e.g. 
```suggestion
      vulnType: vuln.VULN_TYPE,
      solution: vuln.SOLUTION,
      discoveryRemote: vuln.DISCOVERY?.REMOTE,
      category: vuln.CATEGORY,
```

Reply:
This could have been a poor assumption on my part to use those names for both entities.

Reply:
`FIND qualys_vuln` **or** `FIND Vulnerability`

Reply:
Also FYI, see ![Screen Shot 2021-08-03 at 4 12 40 PM](https://user-images.githubusercontent.com/15333061/128079734-61bb9acd-8744-4c1f-b727-c3a6b0316132.png)